Tests/LibWeb/TestCSSIDSpeed.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021, Max Wipfli <max.wipfli@serenityos.org>
```

Comment:
Is this copyright correct?

Reply:
You're right! Fixed it in the recent push.